spec/unleash/client_specification_spec.rb

```diff
@@ -19,51 +19,50 @@
     Unleash.configuration.disable_metrics = true
   end
 
-  if File.exist?(SPECIFICATION_PATH + '/index.json')
-    JSON.parse(File.read(SPECIFICATION_PATH + '/index.json')).each do |test_file|
-      describe "for #{test_file}" do
-        current_test_set = JSON.parse(File.read(SPECIFICATION_PATH + '/' + test_file))
-        context "with #{current_test_set.fetch('name')} " do
-          tests = current_test_set.fetch('tests', [])
-          state = current_test_set.fetch('state', {})
-          state_features = state.fetch('features', [])
-          state_segments = state.fetch('segments', []).map{ |segment| [segment["id"], segment] }.to_h
-          let(:unleash_toggles) { state_features }
-          unleash_client = Unleash::Client.new
+  unless File.exist?(SPECIFICATION_PATH + '/index.json')
```

Comment:
This looks like a big change, it's 99% whitespace. All this does is make the tests fail instead of warn if the client spec isn't present